Assets/MixedRealityToolkit.Providers/UnityAR/Definitions/ArTrackedPose.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.﻿
 
-namespace Microsoft.MixedReality.Toolkit.Experimental.UnityAR
```

Comment:
Also, might be good to throw this change into the release notes for 2.4 as part of this change, since the scaffolding was added in #7319

Reply:
ahh yes! good call